### ISSUE TITLE
[rulename] issue title

### ISSUE THREAD

**Issue by btd1337** [typescript-eslint/typescript-eslint#3893]
It is possible to add the possibility of not validating this rule when the interface has methods (default way used in languages like Java)?

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-definitions.md

I.E.:

```ts
"@typescript-eslint/consistent-type-definitions": ["error", { "type", => "allowInterfaceWithMethods" <= }]
```

This code generates linter failures:

```ts
interface Test {
   bar(): string;

   foo(): string;
}
```

```
Line 2, column 5: ';' expected.
`parseForESLint` from parser `@typescript-eslint/parser` is invalid and will just be ignored
```

**Comment by bradzacher** [comment]
A few things
1) you didn't fill out the issue title
2) you've ***ignored*** the issue template
3) the code does not error - something else in your setup is erroring

Please take the time to fill out a new issue and follow the issue template, filling out all fields and providing an isolated repro.
